zerver/webhooks/sonarqube/view.py

```diff
@@ -0,0 +1,104 @@
+# Webhooks for external integrations.
+
+from typing import Any, Dict, List, Mapping
+
+from django.http import HttpRequest, HttpResponse
+
+from zerver.decorator import webhook_view
+from zerver.lib.request import REQ, has_request_variables
+from zerver.lib.response import json_success
+from zerver.lib.webhooks.common import check_send_webhook_message
+from zerver.models import UserProfile
+
+inverse_operators = {
+    "WORSE_THAN": "should be better or equal to",
+    "GREATER_THAN": "should be less or equal to",
```

Comment:
Change to `should be less than or equal to`

Reply:
Ok